test/lib/rules/no-unsupported-ssr-properties.js

```diff
@@ -351,7 +401,6 @@ testRule('no-unsupported-ssr-properties', {
         {
             code: `
                 import { LightningElement } from 'lwc';
-
                 export default class Foo extends LightningElement {
                   connectedCallback() {
                     this.childNodes.item(0).textContent = 'foo';
```

Comment:
Having some invalid cases tested here would be helpful to check that the PR is working as expected. For example:

```js
if (a && b && c && d) {
  this.template.querySelector('span');
}
```